lbz/boto3_client.py

```diff
@@ -0,0 +1,61 @@
+from functools import cached_property
```

Comment:
Originally, this module was named simply `boto3_.py` and it was done this way to avoid repeating the `client` word too many times.

But if you don't like this convention, we can go with `boto3_broker.py` or `boto3_adapter.py` 😉